tests/components/scrape/__init__.py

```diff
@@ -124,3 +124,10 @@ async def async_update(self, data: bool | None = True) -> None:
             self.data = None
         if self.count == 3:
             self.data = None
+        if self.payload == "test_scrape_timestamp":
+            self.data = (
+                # Default
+                "<div class='current-time'>"
+                "<h1>Current Time:</h1><span class='utc-time'>2022-12-22T13:15:30Z</span>"
+                "</div>"
+            )
```

Comment:
I think it would be better to add this to the bottom of `test_scrape_sensor` and avoid having yet another data sample.